applications/DEMApplication/python_scripts/DEM_benchmarks_class.py

```diff
@@ -2915,9 +2917,7 @@ def ApplyNodalRotation(self, time, dt, modelpart):
                 node.Z = relative_node_coords[2] + centroid[2]
 
                 displacement = [0]*3
-                displacement[0] = node.X - node.X0
-                displacement[1] = node.Y - node.Y0
-                displacement[2] = node.Z - node.Z0
+                displacement[0], displacement[1], displacement[2] = GetDisplacement(node)
```

Comment:
why not just `displacement = GetDisplacement()` ?

Reply:
fixed